### PR TITLE
fix: Add tomli dependency back again.

### PR DESCRIPTION
**Pull request by ruben-arts**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

While using colcon to build our workspace we encountered the following issue:
```
The 'tomli; extra == "toml"' distribution was not found and is required by coverage
```
Following the issue we found that tomli was not installed anymore since `v6.3.2` but this was no change on the pip package so we found that it was the change in the recipe of the `v6.3.2` release.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ requirements:
     - pip
   run:
     - python
+    - tomli
```
Comment (ocefpaf) [inline recipe/meta.yaml:27]: Just a comment like `# extra dep` at the top here will do.

Comment (ruben-arts) [reply recipe/meta.yaml]: :+1:  Done

**Comment by ocefpaf** [comment]
@ruben-arts do you want to add yourself as a [maintainer](https://conda-forge.org/docs/maintainer/updating_pkgs.html) here before I merge?

**Comment by ruben-arts** [comment]
> 

No i'm good, i have no experience with the package itself.

**Comment by ocefpaf** [comment]
> No i'm good, i have no experience with the package itself.

No problem but what you just did here **is** the experience we need :smile:

**Pull request merged**